### PR TITLE
Add a create decision form in a side panel

### PR DESCRIPTION
**Pull request by mikec**

### CODE REVIEW

File: app/components/CreateDecisionMarketForm.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      <label htmlFor="queston">Question:</label>
+      <Field name="question" component="input" type="text" />
+    </div>
+    <div>
+      <Button type="submit">Create Decision</Button>
+    </div>
+  </form>
+))
+
+export default CreateDecisionMarket
```
Comment (jpkcambridge) [inline app/components/CreateDecisionMarketForm.js]: would it make sense to call this "CreateDecisionMarketForm" to be consistent with the file name?

**Comment by jpkcambridge** [comment]
One comment about a name.  Otherwise, looks good to me 👍

File: app/components/CreateDecisionMarketForm.js (hunk 1 above)
Comment (mikec) [reply app/components/CreateDecisionMarketForm.js]: yep good catch!

**Comment by jpkcambridge** [comment]
👍

**Pull request merged**